pkg/bundle/release.go

```diff
@@ -222,34 +223,40 @@ func downloadMirror(secret []byte, rootDir, from string, skipTlS, dryRun bool) e
 	opts := release.NewMirrorOptions(stream)
 
 	opts.From = from
-	opts.ToDir = rootDir
+	opts.ToDir = toDir
 
 	// FIXME(jpower): need to have the user set skipVerification value
 	// If the pullSecret is not empty create a cached context
 	// else let `oc mirror` use the default docker config location
 	if len(secret) != 0 {
 		ctx, err := config.CreateContext(secret, false, skipTlS)
-
 		if err != nil {
-			return nil
+			return assocs, err
 		}
-
 		opts.SecurityOptions.CachedContext = ctx
 	}
 
 	opts.SecurityOptions.Insecure = skipTlS
 	opts.DryRun = dryRun
 
 	if err := opts.Run(); err != nil {
-		return err
+		return assocs, err
 	}
-	return nil
 
+	// There is currently no way to retrieve mappings created by mirror options,
```

Comment:
I'm not aware of a mapping file being created for release images. Can you confirm @afflom @jpower432?

Reply:
You're correct